applications/ecallmgr/src/ecallmgr_fs_acls.erl

```diff
@@ -107,9 +107,40 @@ system_config_acls(Node) ->
         {'error', Error} ->
             lager:warning("error getting system acls : ~p", [Error]),
             kz_json:new();
-        JObj -> JObj
+        JObj -> resolve_system_config_acls(JObj)
     end.
 
+resolve_system_config_acls(JObj) ->
+    kz_json:map(fun resolve_system_config_acls/2, JObj).
+
+resolve_system_config_acls(K, JObj) ->
+    CIDR = kz_json:get_value(<<"cidr">>, JObj),
+    {K, kz_json:set_value(<<"cidr">>, maybe_resolve_cidr(CIDR), JObj)}.
+
+maybe_resolve_cidr(CIDRS)
+  when is_list(CIDRS) ->
+    [maybe_resolve_cidr(CIDR) || CIDR <- CIDRS];
+
```

Comment:
Please remove this blank line, the rest of functions within this module doesn't add blank lines between function clauses.